openedx/openedx-platform
# fix: unresponsive modal in student dashboard view

## Description
This PR fix student LMS dashboard responsiveness issue, specifically in `unenroll-modal` and `email-settings-modal`

Before:
![Screenshot from 2022-12-02 20-08-32](https://user-images.githubusercontent.com/17548619/206964597-cdae6286-0929-4f07-8c1a-632c7692ba33.png)

After:
![Screenshot from 2022-12-02 20-02-07](https://user-images.githubusercontent.com/17548619/206964641-c11ce39d-0f2b-4d0b-9bb2-f8f4731121e5.png)

Currently, the modal use [jquery leanmodal](https://leanmodal.finelysliced.com.au/) which is not responsive. Since this library adds CSS styles to the modal component directly, `!important` property is necessary to override the old non responsive behavior. 

To make sure the modals size doesn't change with existing UI, the `@media` rule is used.

## Supporting information

Github issue: [build-test-release-wg/228](https://github.com/openedx/build-test-release-wg/issues/228)
PR to `open-release/olive.master`:  #31433 

## Testing instructions

1. Go to the LMS > dashboard
2. Check the email setting button on the course card
